Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    parameter_substitution (0.2.0)
+    parameter_substitution (0.2.1)
```

Comment:
I merged a version bump to master today. You're going to want to make this `0.2.2` before we merge this in.